demoapp/demo_app.py

```diff
@@ -38,7 +42,7 @@ def get_mgl_sections_file():
         print("May take a few minutes")
         urllib.request.urlretrieve("https://drive.google.com/file/d/1eYMmxW4gLyh7Zxh8BBJTvMOUNtdbK5B3/view?usp=share_link", "demoapp/all_bills_with_mgl.pq")
```

Comment:
Thanks for hosting this on drive!  Can you add an inline comment explaining how this file was generated (presumably with `extract_mgl_sections.py`?

Reply:
I think this comment has not yet been addressed.

Reply:
I added a comment in line 15. Did you mean somewhere else?

Reply:
Oh, I see it now!  It shows up as line 43 in the current version for me. Thanks for explaining!